install/install.md

```diff
@@ -97,8 +97,9 @@ ERROR: Could not find a version that satisfies the requirement cudf-cu12 (from v
 ERROR: No matching distribution found for cudf-cu12
 ```
 Check the suggestions below for possible resolutions:
+
 - The pip index has moved from the initial experimental release! Ensure the correct `--extra-index-url=https://pypi.nvidia.com`
-- Only Python versions 3.10 and 3.11 are supported
+- Ensure you're using a Python version that RAPIDS supports (compare the values in the [the install selector](#selector) to the Python version reported by `pip --version`).
```

Comment:
I think we want the Python version, not the pip version. Pip "kind of" reports the Python version, too, but it's not easy to parse.
```suggestion
- Ensure you're using a Python version that RAPIDS supports (compare the values in the [the install selector](#selector) to the Python version reported by `python --version`).
```

Reply:
Ok that's fine with me.

I was thinking that your `python` executable is not always guaranteed to be the same interpreter as your `pip` executable (which is why `python -m pip` is often safer).

When I run `pip --version`, I see this:

```text
pip 23.2.1 from /raid/jlamb/miniforge/lib/python3.10/site-packages/pip (python 3.10)
```

That `(python 3.10)` seems pretty clear to me.

Reply:
I don't feel super strongly about this and I think the difference is a minor one. I've accepted this suggestion to keep this moving forward.